include/umf/pools/pool_jemalloc.h

```diff
@@ -14,8 +14,17 @@
 extern "C" {
 #endif
 
+#include <stdbool.h>
 #include <umf/memory_pool_ops.h>
 
+/// @brief Configuration of Jemalloc Pool
+typedef struct umf_jemalloc_pool_params_t {
+    /// Set to true if umfMemoryProviderFree()
+    /// of the chosen provider always fails.
+    /// It is used to turn of the error messages.
+    bool provider_free_always_fails;
```

Comment:
If it is used to turn of the errors then I think "no_error_on_fail" or similar would be better. Additionally, if we know that the free will always fail, then we could add a flag "never_free" and don't call umfMemoryProviderFree at all.

Reply:
Done